core/base/Config.h

```diff
@@ -346,15 +346,9 @@ THE SOFTWARE.
 /// @name namespace ax
 /// @{
 #ifdef __cplusplus
-#    define NS_AX_BEGIN \
-        namespace ax    \
-        {
-#    define NS_AX_END }
```

Comment:
I think these should have been just deprecated and left for backwards compatibility.